scarb/src/core/manifest/summary.rs

```diff
@@ -79,27 +78,10 @@ impl Summary {
                 .version_req(DependencyVersionReq::exact(&cairo_version))
                 .build()
         });
-
-        static TEST_PLUGIN_DEPENDENCY: Lazy<ManifestDependency> = Lazy::new(|| {
```

Comment:
sounds like we can remove targets information from summaries and from registry data now?